src/pages/iou/steps/IOUConfirmPage.js

```diff
@@ -25,23 +75,181 @@ const propTypes = {
 
 const defaultProps = {
     iou: {},
+    onUpdateComment: null,
+    comment: '',
 };
 
-const IOUConfirmPage = props => (
-    <View style={styles.pageWrapper}>
-        <ButtonWithLoader
-            style={[styles.button, styles.w100]}
-            text={`Request $${props.iouAmount}`}
-            isLoading={props.iou.loading}
-            onClick={props.onConfirm}
-        />
-    </View>
-);
+class IOUConfirmPage extends Component {
```

Comment:
So the page is supposed to be a controlled component, managed by IOUModal. 

There's a lot of logic for displaying the participant list, so maybe that should be separated into its own component?